### ISSUE TITLE
Why my train loss equal to nan?

### ISSUE THREAD

**Issue by DaBinGOGOGO** [InsaneLife/dssm#17]
Hi,
    When I run dssm_rnn.py, the train loss always shows nan. Change learning rate, no matter what.
I print out the variables in the model, and the variable embedding in the word_embeddings_layer shows nan for the first time.
   How to deal with it. Thanks!

**Comment by InsaneLife** [comment]
`loss = -tf.reduce_sum(tf.log(hit_prob))`
should add a minimal number like
`loss = -tf.reduce_sum(tf.log(hit_prob + 1e-8))`

**Comment by DaBinGOGOGO** [comment]
The log function is a reason. There is also a case where I will get 0 when calculating the norm.
